src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPrintMembers.cs

```diff
@@ -214,7 +214,7 @@ static bool isPrintable(Symbol m)
                     return false;
                 }
 
-                if (m.Kind is SymbolKind.Field)
+                if (m is SymbolField { IsConst: false })
                 {
                     return true;
```

Comment:
>true [](start = 27, length = 4)

We should exclude `static` fields and properties as well.

Reply:
I have added a check for `IsStatic`. I think IsConst should be redundant now because constant fields are implicitly static?